shared/templates/static/bash/aide_scan_notification.sh

```diff
@@ -0,0 +1,16 @@
+# platform = Red Hat Enterprise Linux 7
+CRONTAB=/etc/crontab
+CRONDIRS='/etc/cron.d /etc/cron.daily /etc/cron.weekly /etc/cron.monthly'
+unset UPDATE_CRONTAB VARSPOOL
+
+if [ -f /var/spool/cron/root ]; then
+	VARSPOOL=/var/spool/cron/root
+fi
+
+if ! grep -qR '^.*\/usr\/sbin\/aide\s*\-\-check.*\|.*\/bin\/mail\s*-s\s*".*"\s*root@.*$' $CRONTAB $VARSPOOL $CRONDIRS; then
+	UPDATE_CRONTAB=1
```

Comment:
ahahaha yeah, this was for "legacy" reasons when I did the first version of the remediation, I'll change that :P